### ISSUE TITLE
New user experience: `hanami new -h`

### ISSUE THREAD

**Issue by backus** [hanami/hanami#782]
I'm trying out Hanami (and liking it) but I wanted to share a small nitpick that made me laugh. I was about to start my first hanami application and I wanted to check what I could specify. Here is what I ended up doing:

```
$ hanami new -h
      create  .hanamirc
      create  .env.development
      create  .env.test
      create  Gemfile
      create  config.ru
      create  config/boot.rb
      create  config/environment.rb
      create  lib/_h.rb
      create  public/.gitkeep
      create  config/initializers/.gitkeep
      create  lib/_h/entities/.gitkeep
      create  lib/_h/repositories/.gitkeep
      create  lib/_h/mailers/.gitkeep
      create  lib/_h/mailers/templates/.gitkeep
      create  spec/_h/entities/.gitkeep
      create  spec/_h/repositories/.gitkeep
      create  spec/_h/mailers/.gitkeep
      create  spec/support/.gitkeep
      create  db/migrations/.gitkeep
      create  Rakefile
      create  spec/spec_helper.rb
      create  spec/features_helper.rb
      create  db/schema.sql
      create  .gitignore
         run  git init . from "."
      create  apps/web/application.rb
      create  apps/web/config/routes.rb
      create  apps/web/views/application_layout.rb
      create  apps/web/templates/application.html.erb
      create  apps/web/assets/favicon.ico
      create  apps/web/controllers/.gitkeep
      create  apps/web/assets/images/.gitkeep
      create  apps/web/assets/javascripts/.gitkeep
      create  apps/web/assets/stylesheets/.gitkeep
      create  spec/web/features/.gitkeep
      create  spec/web/controllers/.gitkeep
      create  spec/web/views/.gitkeep
      insert  config/environment.rb
      insert  config/environment.rb
      append  .env.development
      append  .env.test
```

Not a big deal but seems like it would be nice to handle that better.

**Comment by AlfonsoUceda** [comment]
@backus thanks for reporting it ;) We are working to improve our CLI and make it more robust.

**Comment by jodosha** [comment]
@backus I wanted to inform you that we fixed it in v1.1.0:

```shell
➜ hanami version
v1.1.0.beta3

➜ hanami new -h
Command:
  hanami new

Usage:
  hanami new PROJECT

Description:
  Generate a new Hanami project

Arguments:
  PROJECT             	# REQUIRED The project name

Options:
  --database=VALUE, -d VALUE      	# Database (mysql/mysql2/postgresql/postgres/sqlite/sqlite3), default: "sqlite"
  --application-name=VALUE        	# App name, default: "web"
  --application-base-url=VALUE    	# App base URL, default: "/"
  --template=VALUE                	# Template engine (erb/haml/slim), default: "erb"
  --test=VALUE                    	# Project testing framework (minitest/rspec), default: "minitest"
  --[no-]hanami-head              	# Use Hanami HEAD (true/false), default: false
  --help, -h                      	# Print this help

Examples:
  hanami new bookshelf                     # Basic usage
  hanami new bookshelf --test=rspec        # Setup RSpec testing framework
  hanami new bookshelf --database=postgres # Setup Postgres database
  hanami new bookshelf --template=slim     # Setup Slim template engine
  hanami new bookshelf --hanami-head       # Use Hanami HEAD
```
